docs/kube-up.md

```diff
@@ -0,0 +1,22 @@
+# Production Quality Cluster on GCE
+This document provides the steps to bring up a production quality cluster on GCE with [`kube-up.sh`](https://kubernetes.io/docs/getting-started-guides/gce/).
+
+## Download CRI-Containerd Release Tarball
+To download release tarball, see [step 1](./installation.md#step-1-download-cri-containerd-release-tarball) in installation.md.
+
+Unpack release tarball to any directory, using `${CRI_CONTAINERD_PATH}` to indicate the directory in the doc:
+```shell
+tar -C ${CRI_CONTAINERD_PATH} -xzf cri-containerd-${VERSION}.linux-amd64.tar.gz
+```
+## Set Environment Variables for CRI-Containerd
+Use environment variable `CRI_CONTAINERD_VERSION` to specify `cri-containerd` version. By default,
+latest version will be used.
+```shell
+. ${CRI_CONTAINERD_PATH}/cluster/gce/env
+```
+## Create Kubernetes Cluster on GCE
+Follow the [instruction](https://kubernetes.io/docs/getting-started-guides/gce/) to create a production quality Kubernetes cluster on GCE.
+
+**Please make sure that the Kubernetes version you are using is 1.9+.** To enforce the Kubernetes cluster version:
+* If you are using `https://get.k8s.io`, you could use environment variable `KUBERNETES_RELEASE` to set the version.
```

Comment:
/s/If you are/When/
/s/you could use/use the/

Is there a link to an example here? or provide one..

Reply:
https://get.k8s.io is an odd link to a text file..

Reply:
Yeah, we provide this kind of installation: `curl -sS https://get.k8s.io | bash`.

Reply:
kk yeah that seems important to explain.. otherwise the link is confusing

Reply:
Done.